src/markdown-pages/automate-workflows/get-started-new-relic-cli.mdx

```diff
@@ -32,42 +32,42 @@ Access the New Relic platform from the comfort of your terminal: you can use the
-2. An [instrumented application](https://docs.newrelic.com/docs/agents/manage-apm-agents/installation/install-agent) in your New Relic account
+- Your New Relic [personal API Key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#personal-api-key), which you can create from the **Account settings** of your New Relic account
+- An [instrumented application](https://docs.newrelic.com/docs/agents/manage-apm-agents/installation/install-agent) in your New Relic account
 
 <Steps>
 <Step>
 
 ## Install the New Relic CLI
 
-The New Relic CLI can be downloaded via [Homebrew](https://brew.sh/) (macOS), [Scoop](https://scoop.sh/) (Windows), and [Snapcraft](https://snapcraft.io/) (Linux). You can also download [pre-built binaries](https://github.com/newrelic/newrelic-cli/releases) for all platforms, including .deb and .rpm packages, and our Windows x64 .msi installer.
+Download the New Relic CLI for your operating system, as described below. You can also download [pre-built binaries](https://github.com/newrelic/newrelic-cli/releases) for all platforms, including .deb and .rpm packages, and our Windows x64 .msi installer.
 
-### Linux
+**Linux**
 
-With [Snapcraft](https://snapcraft.io/) installed, run:
+Using [Snapcraft](https://snapcraft.io/), run:
 
 `sudo snap install newrelic-cli`
 
-### macOS
+**macOS**
 
-With [Homebrew](https://brew.sh/) installed, run:
+Using [Homebrew](https://brew.sh/), run:
 
 `brew install newrelic-cli`
 
-### Windows
+**Windows**
 
-With [Scoop](https://scoop.sh/) installed, run:
+Using [Scoop](https://scoop.sh/), run:
 
 `scoop bucket add newrelic-cli https://github.com/newrelic/newrelic-cli.git`<br />
 `scoop install newrelic-cli`
 
 </Step><Step>
 
-## Create your New Relic CLI profile
+Create your New Relic CLI profile.
 
-Now that you've installed the New Relic CLI, it's time to create your first profile. Profiles contain credentials and settings that you can apply to any CLI command, which is useful when switching between accounts.
+After you install the New Relic CLI, it's time to create your first profile. Profiles contain credentials and settings that you can apply to any CLI command, which is useful when switching between accounts.
 
-To create your first CLI profile, run the [`profiles add`](https://github.com/newrelic/newrelic-cli/blob/master/docs/cli/newrelic_profile_add.md) command. Note that you need to set the [region](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers) of your New Relic account: use `-r` to set either `us` or `eu` (this is required).
+Run the [`profiles add`](https://github.com/newrelic/newrelic-cli/blob/master/docs/cli/newrelic_profile_add.md) command. Note that you need to set the [region](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers) of your New Relic account: use `-r` to set either `us` or `eu` (this is required).
```

Comment:
yes. And I don't know what the best option is because the code is to the right of the info. What do you think?

Reply:
I actually created an Issue, suggesting we only render code blocks in the text. I think putting the code blocks beside the text introduces more issues than it solves. So, if it goes that direction, I think colons. If it remains as is, that’s harder, because the colons look bad in the 2-column layout, but good in the 1-column layout. Though, a period rarely feels like a good lead-in, to me, regardless of the arrangement.

Reply:
@jerelmiller solved the layout problem with the new cli/code block UI code. Now, with proper code alignment, I think colons make the most sense, regardless of the 1-column or 2-column layout.